### PR TITLE
Make rpc url

### LINKED ISSUE #751: Make RPC url of scan configurable
Should be through env variable. Also work with Swit to update render.com env variable.

Here's some pointer: https://parceljs.org/env.html

### PR DESCRIPTION
**Pull request by prin-r**
Fix #751

### CODE REVIEW

File: scan/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "parcel-bundler": "^1.12.4"
   },
   "scripts": {
-    "build": "bsb -make-world && parcel build index.html",
+    "build": "rm -rf dist/ && bsb -make-world && parcel build index.html --no-source-maps --no-cache",
```
Comment (sorawit) [inline scan/package.json:9]: Why is this change necessary?

Comment (prin-r) [reply scan/package.json:9]: --no-cache is necessary because sometime it will use the env from cache. So we just remove the cache.
"rm -rf dist/" and "--no-source-maps" are irrelevant to this pr, I just add them to reduce redundant build in dist folder and reduce the size of the bundle.

Comment (sorawit) [reply scan/package.json:9]: Sounds good!

**Review by sorawit — APPROVED** [review]

**Comment by sorawit** [comment]
I did set environment variable in render.com to have `RPC_URL=https://d3n.bandprotocol.com/` and it seems to be working just fine!

**Review by evilpeach — APPROVED** [review]
